app/src/components/pages/Survey.tsx

```diff
@@ -1,7 +1,18 @@
 import React from 'react';
+import SurveyForm from 'components/organisms/SurveyForm';
+import Typography from '@mui/material/Typography';
+import myData from '../../classList.json';

```

Comment:
Relative import should be absolute, can just import using `import myData from 'classList.json'`